src/lib.rs

```diff
@@ -96,6 +96,71 @@
 #![cfg_attr(nightly_docs, deny(broken_intra_doc_links))]
 #![cfg_attr(nightly_docs, feature(doc_cfg))]
 
+/// Defines a mark to be used with [`hit!`] and [`chk!`].
+///
+/// # Example
+///
+/// ```
+/// #[cfg(test)]
+/// mod tests {
+///    cov_mark::def!(no_op);
+/// }
+/// ```
+#[macro_export]
+macro_rules! def {
+    ($ident:ident) => {
```

Comment:
This might want a `#[cfg(test)]` to avoid creating marks when not testing.

Reply:
Not sure about this: shouldn't the `define` go in a `#[cfg(test)]` anyway? Also, I'm not sure on how to write it: is this fine?

```rust
#[macro_export]
#[cfg(not(feature = "thread-local"))]
macro_rules! define {
    ($ident:ident) => {
        #[cfg(test)]
        #[no_mangle]
        static $ident: $crate::__rt::HitCounter = $crate::__rt::HitCounter::new();
    };
}
```

Test runs fine, but I get a bunch of errors in rust-analyzer with this.